include/deal.II/hp/dof_level.h

```diff
@@ -505,8 +607,7 @@ namespace internal
       ar & this->cell_dof_indices_cache;
       ar & this->dof_indices;
       ar & this->dof_offsets;
-      ar & this->p_refine_flags;
-      ar & this->p_coarsen_flags;
+      ar & this->future_fe_indices;
```

Comment:
You asked in a previous comment whether it makes sense to serialize these flags. One can probably think about it in different ways, but it definitely doesn't hurt.

Reply:
I have no idea where that comment went :-)

I decided to keep it, since we do the same for `cell_refine|coarsen_flags` in `TriaLevel`.
https://github.com/dealii/dealii/blob/095d16fdaf8d016141b85f7c43a727192b20ea2b/include/deal.II/grid/tria_levels.h#L272